tests/integration_tests.rs

```diff
@@ -283,8 +339,8 @@ async fn raw() -> Fallible<()> {
     let res = client
         .batch_get(vec!["k4".to_owned(), "k3".to_owned()])
         .await?;
-    assert_eq!(res[0].1, "v4".as_bytes());
-    assert_eq!(res[1].1, "v3".as_bytes());
+    assert_eq!(res[0].1, "v3".as_bytes());
```

Comment:
Because the result is not the same order with the input, it's better to also check the key